### PR TITLE
REMOVES THE ZIZO ZIZZLE!!!!!!!!!!

### PR DESCRIPTION
**Pull request by GuppyLaxx**
This has bothered me since the DAY F***ING ONE.
Christ what a dumb, simple fix.

### CODE REVIEW

**Comment by Hmeister-real** [comment]
it's iconic...

File: code/datums/wounds/types/special.dm [hunk 1]
```diff
@@ -94,6 +94,11 @@
 /datum/wound/facial/eyes/right/permanent
 	whp = null
 	woundpain = 0
+	sound_effect = null
+
+/datum/wound/facial/eyes/right/permanent/on_mob_gain(mob/living/affected)
+	if(mob_overlay)
+		affected.update_damage_overlays()
```
Comment (Cyprex) [inline code/datums/wounds/types/special.dm:101]: Shouldn't this change be added to a parent type?

Comment (Cyprex) [reply code/datums/wounds/types/special.dm:101]: This looks vaguely like it should be handled on /datum/wound or something, I'm not sure what makes sense here!

**Comment by radarseas** [comment]
Zízo

**Comment by ChungusGamer666** [comment]
soul removal

**Pull request merged**